asset-transfer-basic/application-gateway-java/src/main/java/application/java/App.java

```diff
@@ -0,0 +1,251 @@
+               
+        try (Gateway gateway = builder.connect()) {
+        		Network network = gateway.getNetwork(channelName);
+        		Contract contract = network.getContract(chaincodeName);
+        	
+        			//Initialize the Ledger with some data
+        			System.out.println("InitLedger");
+        			initLedger(contract);
+        			System.out.println();
+                    
+                    //getAllAssets
+                    System.out.println("Get All Assets");
+                    getAllAssets(contract);
+                    System.out.println();
+                    //createAsset
+
+                    System.out.println("Create Asset");
+                    createAsset(contract);
+                    System.out.println();
+
+                    //transferAssetAsync
+                    System.out.println("Transfer Asset asynchronously");
+                    transferAssetAsync(contract);
+                    System.out.println();
+
+                    //readAssetByID
+                    System.out.println("Read Asset By ID");
+                    readAssetById(contract);
+                    System.out.println();                    
+
+                    //updateNonExistentAsset
+                    System.out.println("Update a non-existent Asset");
+                    updateNonExistentAsset(contract);
+                    System.out.println();                    
+
+                    
+
+		} catch(Throwable e){
+		  e.printStackTrace();
+		} finally {
+                    channel.shutdownNow().awaitTermination(5, TimeUnit.SECONDS);
+
+                }
+                System.exit(0);
+
+	}
+	private static void initLedger(Contract contract) throws GatewayException, CommitException {
+        
+
+        String timestamp = LocalDateTime.now().toString();
+
+        System.out.println("Submitting \"InitLedger\" transaction with arguments: time, " + timestamp);
+
+        // Submit a transaction, blocking until the transaction has been committed on the ledger.
+        byte[] submitResult = contract.submitTransaction("InitLedger", "time", timestamp);
+
+        System.out.println("Submit result: " + new String(submitResult, StandardCharsets.UTF_8));
+        System.out.println("Evaluating \"InitLedger\" query with arguments: time");
+
+        byte[] evaluateResult = contract.evaluateTransaction("InitLedger", "time");
+        System.out.println("Query result: " + new String(evaluateResult, StandardCharsets.UTF_8));
+    }
+	
+    private static void getAllAssets(Contract contract) throws GatewayException, CommitException {
+
+        String timestamp = LocalDateTime.now().toString();
+        System.out.println("Submitting \"getAllAssets\" transaction with arguments: time, " + timestamp);
+
+        // Submit a transaction, blocking until the transaction has been committed on the ledger.
+        byte[] result = contract.evaluateTransaction("getAllAssets", "time", timestamp);
```

Comment:
There should be no arguments to the GetAllAssets call.

https://github.com/hyperledger/fabric-samples/blob/0d7fa171d146a59dd72bd9e52f2e6e0f8a15fc47/asset-transfer-basic/application-gateway-typescript/src/app.ts#L114-L122